.travis.yml

```diff
@@ -15,6 +19,17 @@ dist: trusty
 matrix:
   allow_failures:
     - php: hhvm
+  exclude:
+    - php: 5.4
+      arch: ppc64le
+    - php: 5.5
+      arch: ppc64le
+    - php: 5.6
+      arch: ppc64le
+    - php: 7.0
+      arch: ppc64le
+    - php: 7.3
+      arch: ppc64le
```

Comment:
If this is included, I'd rather see a single `include` here instead of multiple `exclude` statements (both more obvious and less maintenance overhead once new PHP versions are added).